app/models/grda_warehouse/synthetic/youth_education_status.rb

```diff
@@ -0,0 +1,106 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse::Synthetic
+  class YouthEducationStatus < GrdaWarehouseBase
+    self.table_name = 'synthetic_youth_education_statuses'
+
+    belongs_to :enrollment, class_name: 'GrdaWarehouse::Hud::Enrollment', optional: true
+    belongs_to :client, class_name: 'GrdaWarehouse::Hud::Client', optional: true
+    belongs_to :source, polymorphic: true, optional: true
+    belongs_to :hud_youth_education_status, class_name: 'GrdaWarehouse::Hud::YouthEducationStatus', optional: true, primary_key: :hud_youth_education_status_youth_education_status_id, foreign_key: :YouthEducationStatusID
+
+    validates_presence_of :enrollment
+    validates_presence_of :client
+
+    # Subclasess must define
+    #   information_date, data_collection_stage, data_source
+
+    # Subclasses may override
+    # The HUD spec allows for nil here, but doesn't define what it would mean -- guessing data not collected
+    def current_school_attendance
+      99
+    end
+    alias CurrentSchoolAttend current_school_attendance
+
+    # Should be nil unless current_school_attendance == 0
+    def most_recent_educational_status
+      nil
+    end
+    alias MostRecentEdStatus most_recent_educational_status
+
+    # Should be nil unless current_school_attendance.in?([1, 2])
+    def current_educational_status
+      nil
+    end
+    alias CurrentEdStatus current_educational_status
+
+    def self.hud_sync
+      # Import synthetic events
+      GrdaWarehouse::Synthetic.available_youth_education_status_types.each do |class_name|
+        class_name.constantize.sync
+      end
+
+      create_hud_youth_education_statuses
+
+      # Clean up orphans in HUD table
+      GrdaWarehouse::Hud::YouthEducationStatus.
+        where(synthetic: true).
```

Comment:
We've made this a scope on Event and Assessment.  We should probably do the same here.
```
scope :importable, -> do
      where(synthetic: false)
    end

    scope :synthetic, -> do
      where(synthetic: true)
    end
```